### PR TITLE
Proposal for ROOT/TMVA: 3D convolution on GPU

### PR DESCRIPTION
**Pull request by lmoneta**
A new GSOC proposal for ROOT/TMVA

### CODE REVIEW

File: _gsocproposals/2021/proposal_TMVADLConv3DGPU.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+## Task ideas and expected results
+The initial task of the student will be validating and integrating in ROOT the  CPU implementation for 3D convolution that was developed last summer .
+Afterwards, the main task will be the development of the 3D convolution for Nvidia GPU using the cuDNN library, following the current implementation present in ROOT/TMVA for the 2D operator. 
+The developed code should support both the training and inference for deep learning models.
+
+## Requirements 
+Strong C++ skills, solid knowledge of deep learning, familiarity with GPUs
+
+## Mentors
```
Comment (aflp91) [inline _gsocproposals/2021/proposal_TMVADLConv3DGPU.md:29]: Lorenzo, you need to :  
- add Omar in `gsoc/2021/mentors.md` file
- move your name to the right place (entries must be sorted in **last name** alphabetic order)

**Review by aflp91 — CHANGES_REQUESTED** [review]

File: _gsocproposals/2021/proposal_TMVADLConv3DGPU.md (hunk 1 above)
Comment (lmoneta) [reply _gsocproposals/2021/proposal_TMVADLConv3DGPU.md:29]: I have update the mentor list, 
Cheers
L.

Comment (aflp91) [reply _gsocproposals/2021/proposal_TMVADLConv3DGPU.md:29]: Thanks! But do you want to add Sitong as a mentor in the proposal?

Comment (lmoneta) [reply _gsocproposals/2021/proposal_TMVADLConv3DGPU.md:29]: No, Sitong will not be mentoring this student.

**Review by aflp91 — APPROVED** [review]
Looks OK with me

File: _gsocproposals/2021/proposal_TMVADLConv3DGPU.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  * [Lorenzo Moneta](mailto:Lorenzo.Moneta@cern.ch)
+  * [Omar Zapata](mailto:Omar.Zapata@cern.ch)
```
Comment (agheata) [inline _gsocproposals/2021/proposal_TMVADLConv3DGPU.md:31]: Main mentor must appear in **bold**

**Review by agheata — CHANGES_REQUESTED** [review]

File: _gsocproposals/2021/proposal_TMVADLConv3DGPU.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
+title: TMVA Deep Learning Developments - 3D Convolutions for GPU
+layout: gsoc_proposal
+project: TMVA
+year: 2020
```
Comment (xvallspl) [inline _gsocproposals/2021/proposal_TMVADLConv3DGPU.md:5]: Year should be 2021

**Review by xvallspl — CHANGES_REQUESTED** [review]

File: _gsocproposals/2021/proposal_TMVADLConv3DGPU.md (hunk 3 above)
Comment (lmoneta) [reply _gsocproposals/2021/proposal_TMVADLConv3DGPU.md]: Thanks for spotting this. It is updated now !

**Review by xvallspl — APPROVED** [review]

**Review by agheata — APPROVED** [review]

**Pull request merged**